app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# ApplicationHelper contains all the view helpers.
+module DashboardHelper
+
+  # Render Nodes Table row
+  def nodes_table_row(icon, name, ram, cpu, network, master)
```

Comment:
It's OK for now. In rails there exists the notion of `partials` that we can use here, instead of using the helpers for doing some composition. The idea behind helpers is that they implement logic that would be harder to track and mantain on a view (or something several views can use). Here I'd go with a partial for implementing each row, or even on the same view. We could add a helper for the icon, so the helper method would only include the `case` statement. The rest actually belongs more to a view than to a helper.